charts/opensearch-operator/values.yaml

```diff
@@ -11,3 +11,4 @@ manager:
   image:
     repository: public.ecr.aws/opsterio/opensearch-operator
     tag: latest
+  watchNamespace:
```

Comment:
Could you add a comment here to explain that an empty watchNamespace means all namespaces are watched. Just so we have it documented somewhere.
For the future I think we will need a section in the user guide to explain the available helm values.

Reply:
Will do.

Reply:
Addressed the comment.